resources/profiles/Anker.ini

```diff
@@ -166,16 +178,265 @@ top_solid_layers = 4
 
 [print:0.10 mm HIGHDETAIL (0.4 mm nozzle) @ANKER]
 inherits = *0.10mm*
-compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==0.4
+compatible_printers_condition = printer_notes=~/.*(MACHINE_M5_).*/ and nozzle_diameter[0]==0.4
 
 [print:0.20 mm NORMAL (0.4 mm nozzle) @ANKER]
 inherits = *0.20mm*
-compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==0.4
+compatible_printers_condition = printer_notes=~/.*(MACHINE_M5_).*/ and nozzle_diameter[0]==0.4
 
 
 [print:0.30 mm SUPERDRAFT (0.4 mm nozzle) @ANKER]
 inherits = *0.30mm*
-compatible_printers_condition = printer_model=~/(M5).*/ and nozzle_diameter[0]==0.4
+compatible_printers_condition = printer_notes=~/.*(MACHINE_M5_).*/ and nozzle_diameter[0]==0.4
+
+[print:*common-M5C*]
```

Comment:
the propet way to add this would be to `inherits = *common*` then overwrite the values you want to change.

Reply:
The 3D printer M5C inherits its printing parameters from *common-M5C*, while the M5 inherits its printing parameters from *common*.